spec/spec_helper.rb

```diff
@@ -15,6 +15,9 @@
 Rainbow.enabled = false
 
 require 'haml_lint'
+require 'haml_lint/spec/normalize_indent'
+require 'haml_lint/spec/shared_linter_context'
+require 'haml_lint/spec/matchers/report_lint'
```

Comment:
You could create `lib/haml_lint/spec.rb` to contain these lines and then you would only need:

```ruby
require 'haml_lint/spec'
```

Might be helpful for other consumers as well.

I don't know how folks will use this, but I think it's a safe assumption that if you're `require`ing this that you want to have HAML Lint spec helpers injected into your RSpec runtime.

Reply:
Awesome thanks :D